docs/version0.27.md

```diff
@@ -79,6 +80,10 @@ Global lock reservation is now enabled by default. This is an optimization targe
 
 The default operating system stack size on AIX and Linux PPC64 is increased from 256KB to 512KB. You can change the operating system stack size by using the [-Xmso](xmso.md) option.
 
+### New `-x` option recognized by `jpackcore` / `jextract`
+
+The new option, `-x`, causes the system dump to be omitted from the archive created. In its place, the file `excluded-files.txt` is added which names the excluded file.
+
 ## Full release information
```

Comment:
Would be good to add a link to the jpackcore topic, eg "For more information about this tool, see [Dump extractor (jpackcore)](tool_jextract.md). Also, there should be a comma before "which".

Reply:
I added the comma and a similar sentence, copied from `diag_overview.md`.